lib/engine/game/g_18_chesapeake.rb

```diff
@@ -90,6 +124,16 @@ def cornelius
       def or_set_finished
         depot.export! if %w[2 3 4].include?(@depot.upcoming.first.name)
       end
+
+      def float_corporation(corporation)
+        super
+
+        return unless players.size == 2
```

Comment:
return super unless players

Reply:
I want to call super no matter what. For the order of how logs will come out, I want super called before the special 2p stuff is handled.

Reply:
Would this be better as an `after_float` method?

Reply:
nah, tthis is fine